### ISSUE TITLE
Add new bindings UI

### ISSUE THREAD

**Issue by fredemmott** [OpenKneeboard/OpenKneeboard#522]
The current bindings UI isn't really scalable to lots of actions or lots of devices.

There isn't really a good component to do this in WinUI3; I'm hoping to use the proposed 'table view' control which was planned for v1.5 of the Windows App SDK, but it's now postponed to v1.6, so that's a few months out. Soft-blocked on that: it'll be possible to do something good without it, but potentially a lot more work.

**Comment by fredemmott** [comment]
Component now postponed to v1.7: https://github.com/microsoft/WindowsAppSDK/discussions/4256

**Comment by fredemmott** [comment]
MS still haven't delivered the table view control or equivalent, but I'm moving away from WinUI3. Blocker removed as part of #450
